news/2025/2025-01-25-new-featured-artist-powerwolf.md

```diff
@@ -0,0 +1,57 @@
+
+A total of **14** songs are part of [**Powerwolf**'s Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/457). If you're not picking out a pre-timed `.osz` already, your priorities are out of line.
+
+### [Powerwolf - Christ & Combat](https://assets.ppy.sh/artists/457/Blessed%20and%20Possessed/Powerwolf%20-%20Christ%20%26%20Combat.osz)
+
+We've had **Powerwolf** in our backlog for a long time — since before [COE 2024](/wiki/Community/cavoe's_osu!_event/2024), where [Optimism](https://osu.ppy.sh/users/2064831) blasted through the streams of [Christ & Combat](https://osu.ppy.sh/beatmapsets/1013426) by [piroshki](https://osu.ppy.sh/users/7645522) using only his thumb and index finger. We're not sure how it's possible either, but the proof is here:
+
+<div align="center" class="osu-md__paragraph">
+    <video width="95%" controls>
+        <source src="https://assets.ppy.sh/artists/457/release_showcase.mp4" type="video/mp4" preload="none">
+    </video>
+</div>
+
+### [Powerwolf - Army Of The Night](https://assets.ppy.sh/artists/457/The%20Monumental%20Mass%20-%20A%20Cinematic%20Metal%20Event/Powerwolf%20-%20Army%20Of%20The%20Night.osz)
+
+Try out [this Ranked map](https://osu.ppy.sh/beatmapsets/1044785) also mapped by [piroshki](https://osu.ppy.sh/users/7645522) or [this Loved rendition](https://osu.ppy.sh/beatmapsets/764649) hosted by [Amberessence](https://osu.ppy.sh/users/10145026)!
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/457/The%20Monumental%20Mass%20-%20A%20Cinematic%20Metal%20Event/Powerwolf%20-%20Army%20Of%20The%20Night.mp3">
+</audio>
+
+### [Powerwolf - Incense & Iron](https://assets.ppy.sh/artists/457/The%20Monumental%20Mass%20-%20A%20Cinematic%20Metal%20Event/Powerwolf%20-%20Incense%20%26%20Iron.osz)
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/457/The%20Monumental%20Mass%20-%20A%20Cinematic%20Metal%20Event/Powerwolf%20-%20Incense%20%26%20Iron.mp3">
+</audio>
+
+### [Powerwolf - Blessed & Possessed](https://assets.ppy.sh/artists/457/Blessed%20and%20Possessed/Powerwolf%20-%20Blessed%20%26%20Possessed.osz)
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/457/Blessed%20and%20Possessed/Powerwolf%20-%20Blessed%20%26%20Possessed.mp3">
+</audio>
+
+### [Powerwolf - Beast of Gevaudan](https://assets.ppy.sh/artists/457/The%20Monumental%20Mass%20-%20A%20Cinematic%20Metal%20Event/Powerwolf%20-%20Beast%20of%20Gevaudan.osz)
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/457/The%20Monumental%20Mass%20-%20A%20Cinematic%20Metal%20Event/Powerwolf%20-%20Beast%20of%20Gevaudan.mp3">
+</audio>
+
+---
+
+Even more of **Powerwolf**'s music can be found on [the band's Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/457), pre-timed and ready to be the base for your next map.
```

Comment:
this makes more sense to me, subjective though
```suggestion
Even more of **Powerwolf**'s music can be found on the band's [Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/457), pre-timed and ready to be the base for your next map.
```

Reply:
"Featured Artist listing" alone makes me think of https://osu.ppy.sh/beatmaps/artists

Reply:
subjective